src/ProductRow.js

```diff
@@ -43,6 +43,11 @@ export default class ProductRow extends Component {
   }
 
   render () {
+    const blankCells = this.props.features.map((x) => {
+      return (<td />)
+    })
+    blankCells.push(<td />)
+    // TODO: refactor some amount of this into its own function
```

Comment:
yes, you're right. This is another big `render()` method that should get broken up. No time like the present!